spec/test-data/translations/ios_generate_strings_file_from_code/sample-project/Pods/SomePod/Sources/PodClass1.swift

```diff
@@ -12,10 +16,18 @@ enum PodClass1 {
                                               bundle: Bundle(for: BundleToken.self),
                                               value: "podvalue4, %1$@.",
                                               comment: "Pod key 4, with value, bundle and placeholder.")
-    static let podString5 = NSLocalizedString("pod.key5", tableName: "PodStrings",
+    static let podString5 = NSLocalizedString("pod.key5",
+                                              tableName: "PodStrings",
                                               value: "pod value 5\n"
                                                 + "with multiple lines.",
                                               comment: "Pod key 5, with value, custom table and placeholder.")
+    static let podStringX = PodLocalizedString("pod.custom_l10n_function",
+                                               value: "Using custom macro \n"
+                                                    + "with multiple lines.",
+                                               comment: """
+                                               A string processed via a custom macro
+                                               instead of the standard NSLocalizedString one.
+                                               """)
```

Comment:
I didn't know about routines before looking at this PR. Do you think it would be useful to add a few lines here (or in the action `.rb` source, maybe) to explain them? Or is it safe to assume that a future developer would do as I did, "oh, I don't know about this thing, let me look it up: `man genstrings` `/ routine`."

Reply:
Addressed in d828ea11d5305b69395bf5d6377ac23134015f3f